README.md

```diff
@@ -0,0 +1,24 @@
+<h3 align="center">vacay quiz</h3>
+<h4 align="center"><i>Learning Made Fun</i></h4>
+<p align="center"><a href="https://vacay-quiz.netlify.app">Play quizzes »</a></p>
+
+## Overview
+Vacay quiz is a web application for playing quizzes based on various Indian destinations. Vacay quiz is publicly hosted on Netlify at https://vacay-quiz.netlify.app/.
```

Comment:
UX Improvement - You can use the < a > tag to open the link in new tab, that way, the user does not loose the Github tab.

Reply:
I tried this but it is not supported by github readme.